x-pack/plugins/security_solution/server/lib/detection_engine/fleet_integrations/api/get_installed_integrations/installed_integration_set.ts

```diff
@@ -36,7 +45,7 @@ export const createInstalledIntegrationSet = (): IInstalledIntegrationSet => {
   const addPackagePolicy = (policy: PackagePolicy): void => {
     const packageInfo = getPackageInfoFromPolicy(policy);
     const integrationsInfo = getIntegrationsInfoFromPolicy(policy, packageInfo);
-    const packageKey = `${packageInfo.package_name}:${packageInfo.package_version}`;
+    const packageKey = `${packageInfo.package_name}`;
```

Comment:
As mentioned in the description, I removed `package_version` from the `packageKey` when calculating installed integrations as there can be mis-matches between different policy versions and the integration itself, and I believe the intended behavior here is to not have multiple `relatedIntegrations` returned for different versions (just one for each package+integration combination).